docs/releases.rst

```diff
@@ -11,6 +11,10 @@ tmt-1.42.0
 The ``tmt show`` command now prints in verbose mode manual test
 instructions as well.
 
+Introducing new subpackage ``tmt+ansible``, to pull in Ansible
+requirements for plugins that use Ansible under the hood,
+:ref:`/plugins/prepare/ansible`, :ref:`/plugins/finish/ansible`,
+and :ref:`/plugins/prepare/feature`.
```

Comment:
When rendered, this is a bit confusing to the user:

    Ansible under the hood, ansible, ansible, and feature.

What about mentioning `prepare` and `finish` steps to differentiate between the two `ansible` links?

Reply:
We can do that:

```
`:ref:`prepare/ansible</plugins/prepare/ansible>`, :ref:`finish/ansible</plugins/finish/ansible>`,
and :ref:`prepare/feature</plugins/prepare/feature>`
```